src/Configuration/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
Suggest moving this down into Config.KeyPerFile then eliminating the Configuration/ folder, bringing Config.KeyPerFile up into src/

Reply:
The Directory.Build.props will be removed in the linked PR. Since the only thing valuable in there is the tags property, which can be moved to the csproj. I can move the files up a directory though.

Reply:
Excellent.